volatility3/framework/constants/linux/__init__.py

```diff
@@ -281,3 +281,39 @@
 )
 
 ELF_MAX_EXTRACTION_SIZE = 1024 * 1024 * 1024 * 4 - 1
+
+# net_device_flags was not always an enum, so these hard coded values
```

Comment:
Thanks for the comment!  This is fine, but much better since it's commented!  5:D